src/background/start.js

```diff
@@ -907,6 +907,13 @@ function start(browser) {
         });
     };
 
+    self.playingOnly = function(message, sender, sendResponse) {
```

Comment:
The function name sounds like it is to close all other tabs except the one that is playing. Would suggest to change it  to `closeAudibleTab`.

Reply:
I agree, changing it.